app/models/cms/section.rb

```diff
@@ -66,7 +73,7 @@ def find_or_create!(name, path, options = {})
   end
 
   def full_path
-    path_from_root = parent ? parent.full_path : "/"
+    path_from_root = parent ? parent.full_path : +"/"
     path_from_root.gsub!(/([^\/])$/,"\\1/")
```

Comment:
This hurts eyes a little bit. Fine to leave it like that but maybe makes more sense to avoid modifying the string - e.g. 
```
path_from_root = parent ? parent.full_path.gsub!(/([^\/])$/,"\\1/") : +"/"
```

Reply:
Done